Formula/graphql-cli.rb

```diff
@@ -0,0 +1,52 @@
+require "language/node"
+
+class GraphqlCli < Formula
+  desc "Command-line tool for common GraphQL development workflows"
+  homepage "https://github.com/Urigo/graphql-cli"
+  url "https://registry.npmjs.org/graphql-cli/-/graphql-cli-3.0.14.tgz"
+  sha256 "bf56bbe425795198bf6fcc488f24fe10779327749671b1a331d4137684d8a72d"
+
+  depends_on "node"
+
+  uses_from_macos "expect" => :test
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/".graphqlconfig.yaml").write <<~EOS
+      projects:
+        test:
+          schemaPath: schme.graphql
+    EOS
+
+    script = (testpath/"test.sh")
+    script.write <<~EOS
+      #!/usr/bin/expect -f
```

Comment:
Could this be made to also support the `expect` binary from brew?

Reply:
> Could this be made to also support the `expect` binary from brew?

Can you elaborate a little bit?

Reply:
Use brewed `expect` instead of the system one, if the latter is not available.

Reply:
```suggestion
      #!/usr/bin/env expect -f
```
This should do the trick right?

Reply:
It should, yes. Thanks.

Reply:
Got you! Updating it.